lib/markdown-preview-view.coffee

```diff
@@ -132,14 +132,21 @@ class MarkdownPreviewView extends ScrollView
   renderMarkdown: ->
     @showLoading() unless @loaded
     @getMarkdownSource().then (source) => @renderMarkdownText(source) if source?
+    .catch (reason) =>
```

Comment:
Without this catch the preview pane would endlessly display as loading despite the promise already having been rejected (eg when the file has been deleted).